mne/preprocessing/maxwell.py

```diff
@@ -0,0 +1,530 @@
+# Authors: Mark Wronkiewicz <wronk.mark@gmail.com>
+#          Jussi Nurminen <jnu@iki.fi>
+
+
+# License: BSD (3-clause)
+
+from __future__ import division
+from os import path as op
+import warnings
+import numpy as np
+from scipy.linalg import pinv
+from scipy.misc import factorial
```

Comment:
I think you also need to nest this one, since Travis and I are still getting the `test_import_nesting.py` failure.